src/transferslot.cpp

```diff
@@ -339,16 +361,18 @@ void TransferSlot::doio(MegaClient* client)
                                 }
                                 else
                                 {
-                                    LOG_warn << "MAC verification failed";
+                                    LOG_warn << "MAC verification failed";                                    
                                     transfer->chunkmacs.clear();
                                     return transfer->failed(API_EKEY);
                                 }
                             }
+                            errorcount = 0;
+                            transfer->failcount = 0;
```

Comment:
Check if `failcount` should actually be reset here, since it's the chunk the one that has been sent/received correctly, but `failcount` could already be greater than zero. Reseting the variable would allow to retry the transfer more than once because after a failure there is a chunk transferred successfully.